src/styles/tasks.module.scss

```diff
@@ -4,6 +4,17 @@
     width: 64%;
 }
 
+.taskSkeletonContainer {
+    padding: 2rem 0 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media (max-width: 448px) {
```

Comment:
- what was the reason for adding a media query on `448`? how did we come up with the number `448`?

Reply:
We have taskCardsContainer in [task card list styles](https://github.com/Real-Dev-Squad/website-status/blob/c3a1a0b6da73ca325b4c72b0cab73eff5e96ddeb/src/components/tasks/card/card.module.scss#L398), which has this max width and is used to style the list of cards after we have the task data. Therefore I have used the same value here as well.